apps/src/componentLibrary/tabs/Tabs.tsx

```diff
@@ -0,0 +1,101 @@
+import classnames from 'classnames';
+import React from 'react';
+
+import {ComponentSizeXSToL} from '@cdo/apps/componentLibrary/common/types';
+import _Tab, {TabModel} from '@cdo/apps/componentLibrary/tabs/_Tab';
+import TabPanel from '@cdo/apps/componentLibrary/tabs/TabPanel';
+
+import moduleStyles from './tabs.module.scss';
+
+export interface TabsProps {
+  /** Array of props for Tabs to render */
+  tabs: TabModel[];
+  /** The function that is called when a Tab is clicked */
+  onChange: (value: string) => void;
```

Comment:
Can we set a default behavior that clicking a tab will just open the associated tabpanel? I feel like as a consumer of this component, I wouldn't want to have to write that logic every time I use it? Same with making `selectedTabValue` an optional prop (`initialTabValue`?), and having this component manage what the currently selected value is.

Reply:
I know I've made a similar comment on previous design system components, but I'd love to emphasize simplicity of use in these components where possible.

Reply:
The general idea behind DSCO is to keep components dumb/controlled, so that all the controlled is handled in higher level smart component. With previous components it was better to delegate controls to the container/higher level smart component. (Investigating usages of existing design system components we can see that we need the data from design system components on a higher level(form values, segmented buttons, etc))

Though, I agree that with Tabs it's rather 50/50, since rendering of the tab content is also handled inside of this component. I think we can use something like we use in RadioButtonGroup here:

```
  const handleChange = useCallback(
    (event: React.ChangeEvent<HTMLInputElement>) => {
      setSelectedValue(event.target.value);
      if (onChange) {
        onChange(event);
      }
    },
    [setSelectedValue, onChange]
  );

```

Reply:
Updated the logic, let me know what you think.
Also will appreciate more opinions on this one since not sure which option is better

Reply:
Agree that it's convenient to have the default behavior baked in, but I'm also thinking, should we enforce that consumers only use one or the other? I'm thinking of cases where consumers might provide an onChange handler that updates the selected tab value some other way that conflicts with the internal handler, leading to unexpected behavior. Could we make it so that if an onChange handler is provided, the consumer must also handle the tab switching manually? I could see some potential usefulness of that behavior if a consumer wants to update some data/save state in redux for example. Curious what y'all think

Also, thoughts on renaming `onChange` -> `onTabSelected` or similar?

Reply:
> Agree that it's convenient to have the default behavior baked in, but I'm also thinking, should we enforce that consumers only use one or the other? I'm thinking of cases where consumers might provide an onChange handler that updates the selected tab value some other way that conflicts with the internal handler, leading to unexpected behavior. Could we make it so that if an onChange handler is provided, the consumer must also handle the tab switching manually? I could see some potential usefulness of that behavior if a consumer wants to update some data/save state in redux for example. Curious what y'all think

I would say let's merge as, discuss it more and them make a separate follow up PR for this if needed. (Since it won't break the component logic and some teams already need this component it to be merged)